.github/workflows/e2e_targeted.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
     with:
       kic-image: ${{ needs.choose-image.outputs.image }}
       load-local-image: ${{ inputs.controller-image == '' }}
-      kong-image: kong/kong:amd64-latest
+      kong-image: kong/kong-gateway-dev:nightly
```

Comment:
note (no action required, just fyi): In a separate PR (working on #4013) I intend replace this constant with a `workflow_dispatch` parameter that will have `kong/kong-gateway-dev:nightly` as a default.